data/card_catalogue/upgradeable-1.json

```diff
@@ -395,67 +235,30 @@
     "tier": "gold",
     "details": {
       "rules": "All players will critically succeed on their next d20 roll this turn.",
-      "flavour": "And the light shone bright upon the chosen.",
+      "flavour": "The light will shine bright upon the chosen.",
       "image": "assets/core/card/illustration/Wish.png"
     }
   },
-  {
-    "title": "Distract",
-    "upgrade": "Lie",
-    "tier": "blue",
-    "details": {
-      "rules": "Target NPC fails their current action and loses focus on you",
-      "flavour": "\"Ok one sec my homework is in my bag I'll grab it in a minute. How was your weekend?\" - Little Sniver",
-      "image": "assets/core/card/illustration/Distract.png"
-    }
-  },
-  {
-    "title": "Lie",
-    "upgrade": "Disguise",
-    "tier": "green",
-    "details": {
-      "rules": "Target NPC will believe your next statement.",
-      "flavour": "\"Yeah of course I did my homework\" - Little Sniver",
-      "image": "assets/core/card/illustration/Lie.png"
-    }
-  },
-  {
-    "title": "Disguise",
-    "upgrade": "Revise",
-    "tier": "red",
-    "details": {
-      "rules": "Change your appearance to match that of another exactly for as long as you'd like.",
-      "flavour": "\"I'm Revins, Sniver went into the other room to get his homework.\" - Little Sniver",
-      "image": "assets/core/card/illustration/Disguise.png"
-    }
-  },
-  {
-    "title": "Revise",
-    "upgrade": "",
-    "tier": "gold",
-    "details": {
-      "rules": "Rewrite the contents of a visible character's memory",
-      "flavour": "\"I'm at the end of my rope. I am done with homework. I will simply erase homework from the memory of the people\" - Little Sniver",
-      "image": "assets/core/card/illustration/Revise.png"
-    }
-  },
   {
     "title": "Cure",
-    "upgrade": "Restore",
+    "upgrade": "Field Medicine",
     "tier": "blue",
     "details": {
-      "rules": "Heal 8 to a visible character",
-      "flavour": "\"Yeah that feels a little better\" - Hugo, Recently Decapitated",
+      "rules": "Cure the ailment of a visible character, or heal them 8.",
+      "flavour": "Treats an upset stomach, a broken heart, or a pierced lung",
       "image": "assets/core/card/illustration/Cure.png"
     }
   },
   {
-    "title": "Restore",
+    "title": "Field Medicine",
     "upgrade": "Renew",
     "tier": "green",
     "details": {
-      "rules": "Heal 10 to a visible character",
-      "flavour": "\"Well that sealed up the wound but it isn't really helping\" - Hugo, Recently Decapitated",
+      "rules": {
+        "active": "Heal 15 to a visible character. They can move twice as fast for the remainder of the session",
+        "passive": "Run twice as fast while carrying friendly characters" 
```

Comment:
IMO, healing needs a lot of help to be a fun mechanic anybody will do willingly. I'm trying to go down the 'wacky augmentation' route.

Reply:
This is hilarious, I like this direction. Makes me think we should offer multiple similar effect + heal cards to make the supporting role more attractive too

Reply:
Yeah I think we could make an entire series of wacky  healbuffs

Reply:
Made an issue: https://github.com/elliottomlinson/rpcg/issues/82